java/fury-core/src/main/java/org/apache/fury/ThreadLocalFury.java

```diff
@@ -265,6 +267,16 @@ public ClassLoader getClassLoader() {
     return bindingThreadLocal.get().getClassLoader();
   }
 
+  @Override
+  public ClassChecker getClassChecker() {
+    return bindingThreadLocal.get().get().getClassResolver().getClassChecker();
+  }
+
+  @Override
+  public SerializerFactory getSerializerFactory() {
```

Comment:
We should not provide this api. It may return different object when classloader changes or we invoke It from another thread

Reply:
You are correct; I will proceed with the corrections